tools/doc/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Internal tool for generating Node.js API docs",
   "version": "0.0.0",
   "engines": {
-    "node": ">=12.10.0"
+    "node": ">=14.0.0"
```

Comment:
Do you plan on the module pages changes being backported to 12.x? If so, you’ll probably want to try to make this PR compatible with it.

Reply:
Fair enough, also that should fix the failing CI